config/config.js

```diff
@@ -4,7 +4,7 @@ export default {
   // Note: typeof(process.env.MOCK) === string
   define: {
     MOCK_UI: process.env.MOCK === 'true',
-    ENDPOINTS_ENV: process.env.MOCK === 'true' ? 'endpoints.mock.js' : 'endpoints.js',
+    ENDPOINTS_ENV: process.env.MOCK === 'true' ? '/mock/endpoints.mock.json' : '/api/v1/endpoints',
```

Comment:
Oops; `ENDPOINTS_ENV` is no longer used or useful; I should remove it.

Reply:
Why are we not providing a "mock" of `/api/v1/endpoints` instead of allowing it to pass through?

Reply:
No longer a relevant comment.